requirements-dev.txt

```diff
@@ -3,5 +3,6 @@ git+https://github.com/asdf-format/asdf-standard
 git+https://github.com/asdf-format/asdf-transform-schemas
 git+https://github.com/asdf-format/asdf-coordinates-schemas
 git+https://github.com/asdf-format/asdf-wcs-schemas
-git+https://github.com/astropy/astropy
+astropy>=0.0.dev0
 git+https://github.com/astropy/asdf-astropy
+numpy>=0.0.dev0
```

Comment:
This isn't quite right, see https://github.com/spacetelescope/romancal/pull/695#discussion_r1189076034 and https://github.com/spacetelescope/romancal/pull/695#discussion_r1189078751 for details.